src/tips/en/find-the-offset-parent-of-an-element.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-09-13
+authors: Kilian Valkhof
+title: Find the offset parent of an element
+tags: ["css", "browser:polypane"]
+---
+
+To know what an element is offset against (for example with `inset`) you need to know its **offset parent**. 
```

Comment:
I suggest expanding the "for example with inset" part to a more complete code example. It might be that people reading this tip don't know what inset is, and where they might use it.